include/field_map_object.h

```diff
@@ -42,7 +42,7 @@ struct MapObjectManager { //declared field_system.h
     u8 unk18[4];
     u8 unk1C[0x124-0x1C];
     LocalMapObject* objects;
-    FieldSystem* fsys;
+    FieldSystem* fieldSystem;
```

Comment:
Im gonna be annoyed at this but if everyone else if fine with it then I'll just grumble